bin/ged2gwb/ged2gwb.ml

```diff
@@ -2694,8 +2694,23 @@ let find_lev0 (strm__ : _ Stream.t) =
   in
   bp, r1, r2
 
-let pass1 gen fname =
+let open_in_bin_with_bom_check fname =
+  let in_channel_has_bom ic =
+    try
+      Char.code (input_char ic) = 239
```

Comment:
You can use `input_char ic = '\239`, which is more readable

Reply:
thank you